### ISSUE TITLE
Upgrade cert-manager to >=1.6.0

### ISSUE THREAD

**Issue by glrf** [projectsyn/component-cert-manager#50]
## Context
We should upgrade to the latest cert-manager version.

Upgrading to cert-manager 1.6 is arguable a breaking change, as [it deprecates old cert-manager resources](https://cert-manager.io/docs/release-notes/release-notes-1.6/). There could be issues for [clusters that ran cert-manager < 1.0](https://cert-manager.io/docs/installation/upgrading/upgrading-1.5-1.6/).

**Comment by megian** [comment]
This becomes more important as OpenShift 4.9 is [not supported](https://cert-manager.io/docs/installation/supported-releases/) with cert-manager 1.5.4.

**Comment by bastjan** [comment]
Stored version on master has been `v1` since component release `v2.0.0`. We don't need any conversion.

```yaml
❯ git checkout v2.0.0                   
[...]
❯ make test                                                                                                                                 
[...]
❯ yq '.spec.versions[] | select(.storage == true) | .name ' compiled/cert-manager/cert-manager/01_helmchart/cert-manager/templates/crds.yaml
v1
---
v1
---
v1
---
v1
---
v1
---
v1
```

**Comment by bastjan** [comment]
Fixed by #66
